src/Agent.Worker/Build/FileContainerServer.cs

```diff
@@ -144,6 +149,8 @@ private async Task<UploadResult> ParallelUploadAsync(IAsyncCommandContext contex
                 return uploadResult;
             }
 
+            var uploadToBlob = String.Equals(context.GetVariableValueOrDefault("agent.LogToBlobstorageService"), "true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
ArtifactToBlobstorageService?

Reply:
Also it looks like this result is only used in UploadAsync, which gets the context.  Thoughts on moving the check there and removing the param?

Reply:
Oh oops wrong FF